content/it/docs/reference/glossary/controller.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Controller
+id: controller
+date: 2018-04-12
+full_link: /docs/concepts/architecture/controller/
+short_description: >
+  Un circuito di controllo che osserva lo stato condiviso del cluster attraverso l'API server and apporta le modifiche necessarie per portate lo stato corrente verso lo stato desiderato.
+
+aka: 
+tags:
+- architecture
+- fundamental
+---
+In Kubernetes, i _controllers_ sono circuiti di controllo che osservano lo stato del {{< glossary_tooltip term_id="cluster" text="cluster">}}, e apportano o richiedono modifiche quando necessario. Ogni _controller_ prova a portare lo stato corrente del cluster verso lo stato desiderato.
```

Comment:
```suggestion
In Kubernetes, i _controllers_ sono control loop che osservano lo stato del {{< glossary_tooltip term_id="cluster" text="cluster">}}, e apportano o richiedono modifiche quando necessario. Ogni _controller_ prova a portare lo stato corrente del cluster verso lo stato desiderato.
```

Reply:
not changed due to previous translation agreement